net-libs/rtrlib/files/rtrlib-0.8.0-cmake4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Please send these upstream where upstream is still alive (and add a link at the top).